tests/rules/no-cross-app-dependencies.js

```diff
@@ -0,0 +1,51 @@
+"use strict";
+const sinon = require('sinon');
+const bundleUtils = require('../../src/util/bundle_utils');
+sinon.stub(bundleUtils, "getAllAppBundleNames").returns(['phoenix', 'another-bundle']);
+
+const rule = require("../../src/rules/no-cross-app-dependencies");
+const RuleTester = require("eslint").RuleTester;
+
+
+// Helpers
+// ----------------------------------------------------------------------------
+
+function addValidCodeTestOptions(code, filename) {
+  return {
+    code: code,
+    parserOptions: { ecmaVersion: 6, sourceType: "module" },
+    filename
+  };
+}
+
+function addInvalidCodeTestOptions(code, filename) {
+  return {
+    code: code,
+    parserOptions: { ecmaVersion: 6, sourceType: "module" },
+    filename,
+    errors: [
+      {
+        message: /Avoid cross app imports if possible/,
+      }
+    ]
+  };
+}
+
+// Tests
+// ----------------------------------------------------------------------------
+
+const bundleFileName = "/Users/cliu/base/coursera/web/static/bundles/phoenix/MyPhoenixComponent.jsx";
```

Comment:
do these need to be absolute paths?
```suggestion
const bundleFileName = "~/base/coursera/web/static/bundles/phoenix/MyPhoenixComponent.jsx";
```